src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callback = compareFunction;
+
+    if (!compareFunction) {
+      callback = (arg1, arg2) => {
+        return '' + arg1 > '' + arg2 ? 1 : -1;
+      };
```

Comment:
Move callback logic to the separate function take it outside the current function and pass it as the default function parameter.
Also you need to rewrite this function like this
<img width="631" alt="image" src="https://github.com/mate-academy/js_array-method-sort/assets/64750958/90408886-606c-4da6-9b68-915c0ea6c6ae">